besu/src/main/java/org/hyperledger/besu/services/TransactionSelectionServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.services;
+
+import org.hyperledger.besu.plugin.services.TransactionSelectionService;
+import org.hyperledger.besu.plugin.services.txselection.TransactionSelectorFactory;
+
+import java.util.Optional;
+
+/** The Storage service implementation. */
+public class TransactionSelectionServiceImpl implements TransactionSelectionService {
```

Comment:
I think it's a good idea to annotate it with @`Unstable` as we have for the other new services

Reply:
Done